src/CachePoolTest.php

```diff
@@ -394,6 +380,7 @@ public function testDeferredSaveWithoutCommit()
         $this->cache->saveDeferred($item);
 
         $this->cache = null;
+        gc_collect_cycles();
```

Comment:
Then why not document this? Documentation for tricky things like this does not belong on GitHub, but in the code.